applications/prompt-proto-service-lsstcomcam/README.md

```diff
@@ -15,11 +15,11 @@ Prompt Proto Service is an event driven service for processing camera images. Th
 | prompt-proto-service.additionalVolumeMounts | list | `[]` | Kubernetes YAML configs for extra container volume(s). Any volumes required by other config options are automatically handled by the Helm chart. |
 | prompt-proto-service.affinity | object | `{}` | Affinity rules for the prompt processing pods |
 | prompt-proto-service.alerts.server | string | `"usdf-alert-stream-dev-broker-0.lsst.cloud:9094"` | Server address for the alert stream |
-| prompt-proto-service.alerts.topic | string | None, must be set | Topic name where alerts will be sent |
+| prompt-proto-service.alerts.topic | string | `""` | Topic name where alerts will be sent |
```

Comment:
This commit of doc update should be squashed with the commit where you update the values